lib/bolt/bolt_option_parser.rb

```diff
@@ -61,6 +61,9 @@ def get_help_text(subcommand, action = nil)
           { flags: OPTIONS[:global],
             banner: GROUP_HELP }
         end
+      when 'guide'
+        { flags: OPTIONS[:global],
```

Comment:
Hm,  I don't think the extra flags are doing any harm here, but it does seem like `--help` is the only flag you could *actually* use with this command. We might eventually want to separate those out.

Reply:
Actually `--format` is also valid for this, which *isn't* currently listed.

Reply:
Er, I think this still needs to be addressed.  Once it is I'm :+1:

Reply:
D'oh. Updated.